### PR TITLE
MKM 10 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/fugitive_codebreaker.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Fugitive Codebreaker
+ManaCost:1 R
+Types:Creature Goblin Rogue
+PT:2/1
+K:Prowess
+K:Haste
+K:Disguise:5 R::This cost is reduced by {1} for each instant and sorcery card in your graveyard:X
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/fugitive_codebreaker.txt:7]: Can't get the reducecost to work

File: forge-gui/res/cardsfolder/upcoming/cryptic_coat.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Cryptic Coat
+ManaCost:2 U
+Types:Artifact Equipment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCloak | TriggerDescription$ When CARDNAME enters the battlefield, cloak the top card of your library, then attach CARDNAME to it. (To cloak a card, put it onto the battlefield face down as a 2/2 creature with ward {2}. Turn it face up any time for its mana cost if it’s a creature card.)
+SVar:TrigCloak:DB$ Cloak | Defined$ ValidLibrary Card.TopLibrary+OwnedBy You | RememberCloaked$ True | SubAbility$ DBAttach
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cryptic_coat.txt:5]: how about just using `TopOfLibrary`?

File: forge-gui/res/cardsfolder/upcoming/cryptic_coat.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+SVar:DBAttach:DB$ Attach | Defined$ Remembered | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 1 | Description$ Equipped creature gets +1/+0 and can't be blocked.
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.EquippedBy | Secondary$ True | Description$ Equipped creature gets +1/+0 and can't be blocked.
+A:AB$ ChangeZone | Cost$ 1 U | Origin$ Battlefield | Destination$ Hand | Defined$ Self | SpellDescription$ Return CARDNAME to it's owner's hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cryptic_coat.txt:10]: typo

File: forge-gui/res/cardsfolder/upcoming/eliminate_the_impossible.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Eliminate the Impossible
+ManaCost:1 U
+Types:Instant
+A:SP$ Investigate | SubAbility$ DBDebuff | SpellDescription$ Investigate. Creatures your opponents control get -2/-0 until end of turn. If any of them are suspected, they're no longer suspected. (To investigate, create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
+SVar:DBDebuff:DB$ PumpAll | ValidCards$ Creature | NumAtt$ -2 | SubAbility$ DBAlterAtribute
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eliminate_the_impossible.txt:5]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/essence_of_antiquity.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Essence of Antiquity
+ManaCost:3 W W
+Types:Artifact Creature Golem
+PT:1/10
+K:Disguise: 2 W
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/essence_of_antiquity.txt:5]: spacing

File: forge-gui/res/cardsfolder/upcoming/hunted_bonebrute.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name:Hunted Bonebrute
+ManaCost:2 B
+Types:Creature Skeleton Beast
+PT:6/2
+K:Menace
+K:Disguise:1 B
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ CARDNAME enters the battlefield, target opponent creates two 1/1 white Dog creature tokens.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hunted_bonebrute.txt:7]: description

File: forge-gui/res/cardsfolder/upcoming/hunted_bonebrute.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+SVar:TrigToken:DB$ Token | TokenAmount$ 2 | TokenScript$ w_1_1_dog | ValidTgts$ Opponent | TokenOwner$ Targeted
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigLoseLife | TriggerDescription$ When CARDNAME dies, each opponent loses 3 life.
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ TriggeredCardOpponent | LifeAmount$ 3
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hunted_bonebrute.txt:10]: why such a weird Defined?

File: forge-gui/res/cardsfolder/upcoming/frantic_scapegoat.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
+Name:Frantic Scapegoat
+ManaCost:R
+Types:Creature Goat
+PT:1/1
+K:Haste
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSuspect | TriggerDescription$ When CARDNAME enters the battlefield, suspect it. (It has menace and can't block.)
+SVar:TrigSuspect:DB$ AlterAttribute | Defined$ Self | Attributes$ Suspected
+T:Mode$ ChangesZoneAll | ValidCards$ Creature.Other+YouCtrl | IsPresent$ Card.Self+IsSuspected | Destination$ Battlefield | TriggerZones$ Battlefield | Execute$ TrigChoose | TriggerDescription$ Whenever one or more other creatures enter the battlefield under your control, if CARDNAME is suspected, you may suspect one of the other creatures. If you do, CARDNAME is no longer suspected.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/frantic_scapegoat.txt:8]: it should still use the Defined condition for LKI

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/frantic_scapegoat.txt]: noresolvingcheck is wrong

**Review by tool4ever — APPROVED** [review]

**Pull request merged**